lambdas/webhook_processor/webhook_processor.py

```diff
@@ -164,12 +434,73 @@ def lambda_handler(event, _context):
         payload = data_object.get('body')
         logger.debug("payload is: '{}'".format(payload))
 
+        # Get the sha and jobs list (if applicable), as well as
+        # the max number of retries for sending the hook.
+        event_type = headers.get('X-GitHub-Event')
+        sha, jobs_list = _parse_hook_for_testing_info(payload, event_type)
+        num_tries = _get_num_tries()
+
         # Send it off!
-        # Save up the results for later processing rather than letting
-        # the errors get raised.
+        # If the url is a known jenkins instance (in constants.py),
```

Comment:
We only want to check against build jenkins tho, no? We don't kick off jobs on test-jenkins for platform.